mesh/mesh.hpp

```diff
@@ -1409,6 +1411,13 @@ class Mesh
    /// should not be deleted by the caller.
    static FiniteElement *GetTransformationFEforElementType(Element::Type);
 
+   /** @brief For the edge or face of a boundary element with the orientation
+       @a o (2D or 3D), return the transformation of the boundary element
+       integration point @ ip to the face element. Supports both internal and
+       external boundaries. */
```

Comment:
Update doxygen that this now works in 1D as well.

It's also a good idea to add a note that in 2D this method uses orientations 0/1 (as returned by `GetBdrElementFace`), and not +1/-1.

Reply:
[2d83759](https://github.com/mfem/mfem/pull/3943/commits/2d83759e49f74d261ffeb41091f6db24111fe34c)